src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,219 @@
+        String query =
+                "SELECT c.id id, c.model model, "
+                        + "m.id manufacturer_id, m.name manufacturer, "
+                        + "m.country country "
+                        + "FROM cars c"
+                        + "INNER JOIN cars_drivers c_d "
+                        + "ON cars.id = cars_drivers.car_id "
+                        + "INNER JOIN manufacturers m "
+                        + "ON manufacturers.id = cars.manufacturer_id "
+                        + "WHERE driver_id = ? AND cars.is_deleted = FALSE "
+                        + "AND drivers.is_deleted = FALSE";
+        List<Car> listOfCars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            preparedStatement.setLong(1, driverId);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                listOfCars.add(parseCarFromResultSet(resultSet));
+            }
+        } catch (SQLException exc) {
+            throw new DataProcessingException("Couldn't get all cars by driver id" + driverId, exc);
+        }
+        return listOfCars;
+    }
+
+    private Car parseCarFromResultSet(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        car.setId(resultSet.getObject("id", Long.class));
+        car.setModel(resultSet.getString("model"));
+        Manufacturer manufacturer = parseManufacturerFromResultSet(resultSet);
+        car.setManufacturer(manufacturer);
+        return car;
+    }
+
+    private Manufacturer parseManufacturerFromResultSet(ResultSet resultSet) throws SQLException {
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setId(resultSet.getObject("manufacturer_id", Long.class));
+        manufacturer.setName(resultSet.getString("manufacturer"));
+        manufacturer.setCountry(resultSet.getString("country"));
+        return manufacturer;
+    }
+
+    private void insertDrivers(Car car) {
+        String insertQuery = "INSERT INTO cars_drivers(car_id, driver_id) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(insertQuery)) {
+            preparedStatement.setLong(1, car.getId());
+            for (Driver driver : car.getDrivers()) {
+                preparedStatement.setLong(2, driver.getId());
+                preparedStatement.execute();
+            }
+        } catch (SQLException exc) {
+            throw new DataProcessingException("Couldn't insert drivers to "
+                    + car + ". ", exc);
+        }
+    }
+
+    private void setCarDrivers(Car car) {
+        String query = "SELECT id, name, license_number "
+                + "FROM cars_drivers "
+                + "INNER JOIN drivers "
```

Comment:
```suggestion
                + "FROM cars_drivers c_d "
                + "INNER JOIN drivers d "
```
use these aliases in query